### DISCUSSION TITLE
Empty target tags are deleted in XLIFF

### DISCUSSION THREAD

**Discussion by rsr-cs** [WeblateOrg/weblate#7173] [Q&A]
When deleting a translated text in Weblate the target-tag is completely removed in the XLIFF files. This causes errors when further processing the files with other tools that require a target tag. I saw that this is by design and was a fix of issue #6927 

I think it would be better if this behaviour could be configured project and system wide.

**Comment by nijel** [comment]
Removal of `<target/>` by https://github.com/WeblateOrg/weblate/commit/42d052c14c784ce6e19ad7200f4e331acd607e9d only happens when creating new translation files within Weblate, not when translating.

In 08d21301c8bb25174b35184209edd07723060c65 I've made a change to not modify strings which are not translated, so if you need specific behavior, you can provide [Template for new translations](https://docs.weblate.org/en/latest/admin/projects.html#template-for-new-translations), and it will be used as is in the 4.11 release.

**Reply by rsr-cs** [reply]
I also experience this in existing XLIFFs and can reproduce in dfiferent projects (and can show you in our environment :-):

1. Delete a target translation
2. Save the translation
3. Commit and push to Git repo
4. The target tag is completely removed

**Reply by nijel** [reply]
Aren't you using [Remove blank strings add-on](https://docs.weblate.org/en/latest/admin/addons.html#remove-blank-strings)? It is expected to cause this.
